mm2src/lp_bot/simple_market_maker.rs

```diff
@@ -0,0 +1,266 @@
+use crate::mm2::lp_bot::{SimpleCoinMarketMakerCfg, SimpleMakerBotRegistry, TickerInfosRegistry, TradingBotContext};
+use crate::mm2::lp_ordermatch::{retrieve_my_maker_orders, MakerOrder};
+use common::{executor::{spawn, Timer},
+             log::{error, info},
+             mm_ctx::MmArc,
+             mm_error::MmError,
+             slurp_url, HttpStatusCode};
+use derive_more::Display;
+use http::{HeaderMap, StatusCode};
+use serde_json::Value as Json;
+use std::collections::HashSet;
+use std::{str::Utf8Error,
+          sync::atomic::{AtomicBool, Ordering}};
+use uuid::Uuid;
+
+// !< constants
+const KMD_PRICE_ENDPOINT: &str = "https://prices.komodo.live:1313/api/v1/tickers";
+
+// !< Type definitions
+pub type StartSimpleMakerBotResult = Result<StartSimpleMakerBotAnswer, MmError<StartSimpleMakerBotError>>;
+pub type StopSimpleMakerBotResult = Result<StopSimpleMakerBotAnswer, MmError<StopSimpleMakerBotError>>;
+
+#[allow(dead_code)]
+#[derive(Deserialize)]
+pub struct StartSimpleMakerBotRequest {
+    cfg: SimpleMakerBotRegistry,
+}
+
+#[cfg(test)]
+impl StartSimpleMakerBotRequest {
+    pub fn new() -> StartSimpleMakerBotRequest {
+        return StartSimpleMakerBotRequest {
+            cfg: Default::default(),
+        };
+    }
+}
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub struct StopSimpleMakerBotAnswer {
+    result: String,
+}
+
+#[cfg(test)]
+impl StopSimpleMakerBotAnswer {
+    pub fn get_result(&self) -> String { return self.result.clone(); }
+}
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub struct StartSimpleMakerBotAnswer {
+    result: String,
+}
+
+#[cfg(test)]
+impl StartSimpleMakerBotAnswer {
+    pub fn get_result(&self) -> String { return self.result.clone(); }
```

Comment:
`return` is not needed here. Clippy doesn't catch it because it doesn't cover tests code.